http4s/build.sbt

```diff
@@ -39,16 +44,27 @@ val `http4s-client` =
       version := "5.0.0+n",
       versionPolicyIntention := Compatibility.None,
       libraryDependencies ++= Seq(
-        ("org.http4s" %% "http4s-client" % http4sVersion).cross(CrossVersion.for3Use2_13),
-        ("org.http4s" %% "http4s-async-http-client" % http4sVersion % Test).cross(CrossVersion.for3Use2_13)
+        ("org.http4s" %%% "http4s-client" % "0.23.6").cross(CrossVersion.for3Use2_13)
```

Comment:
http4s cross-builds for Scala.js as of 0.23.5, so I hard-coded it in here for now. Ideally you should update the version globally but I personally can't chase down the various compiler warnings etc. :)

Also, `CrossVersion.for3Use2_13` is really terrible, I hope you are not publishing any of these modules for Scala 3 like this. It's why we had to bump scalajs-dom to v2.0.0:
- https://github.com/scala-js/scala-js-dom/issues/451

Reply:
I would prefer it if we can manage only one version of http4s. But we need someone to help and address the migration warnings to upgrade to 0.23.6 on the server-side.

> Also, `CrossVersion.for3Use2_13` is really terrible, I hope you are not publishing any of these modules for Scala 3 like this. It's why we had to bump scalajs-dom to v2.0.0:

We don’t publish Scala 3 modules. But are you saying that we could remove the `for3Use2_13` from our http4s dependencies? That would be great.

Reply:
Yes, http4s 0.23.x has been published for Scala 3 from the beginning AFAIK. So in theory you could remove that `for3Use2_13`

FWIW I also tried removing that, but got into a mess of conflicting dependencies probably because `for3Use2_13` is used in many places over your project (I understand it's necessary for Akka). I'll try it again, but no promises :)

> But we need someone to help and address the migration warnings to upgrade to 0.23.6 on the server-side.

I'm not familiar with blaze server, but if it's okay with you I can try and move you to ember server, which in general more emphasis will be placed on going forward anyway.

Reply:
Ok, I think I was able to bump http4s to 0.23.6 and address the warnings without switching to ember. I also tried removed `for3Use2_13` but lets see what CI thinks.